src/main/java/pl/tomaszdziurko/jvm_bloggers/JvmBloggersConfiguration.java

```diff
@@ -33,4 +34,8 @@ public Client getMailingRestClient(@Value("${mailing.apiKey}") String malingApiK
         return client;
     }
 
+    @Bean
+    public CacheManager cacheManager() {
+        return new GuavaCacheManager();
```

Comment:
With ConcurrentHashMap and own implementation you are writing your own cache instead of using already existing one :)